apiStar_folder/templates/index1.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <body>
+
+        <h2>Textarea for user's input</h2>
+        <p>Type input in env.spec format.</p>
+        <textarea id="myTextarea"></textarea>
```

Comment:
Change the `id` of the text area please to something more semantic (e.g. `env-spec-input`).